index.html

```diff
@@ -8,62 +8,194 @@
+			</div>
+            <div class="project-card">
+			  <span class="pjt-card-header heading">Profesional Art Printing Data</span>
+              <span class="pjt-card-sub-header sub-heading">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</span>
+			  <div >
+				<ul class="stack-list">
+					<li><a href="#">html</a></li>
+					<li><a href="#">bootstrap</a></li>
+					<li><a href="#">Ruby</a></li>
+				</ul>
+			  </div>
+			  <button class="pjt-button">See Project</button>
+			</div>
+            <div class="project-card">
+			  <span class="pjt-card-header heading">Profesional Art Printing Data</span>
+              <span class="pjt-card-sub-header sub-heading">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</span>
+			  <div >
+				<ul class="stack-list">
+					<li><a href="#">html</a></li>
+					<li><a href="#">bootstrap</a></li>
+					<li><a href="#">Ruby</a></li>
+				</ul>
+			  </div>
+			  <button class="pjt-button">See Project</button>
+			</div>
+            <div class="project-card">
+			  <span class="pjt-card-header heading">Profesional Art Printing Data</span>
+              <span class="pjt-card-sub-header sub-heading">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</span>
+			  <div >
+				<ul class="stack-list">
+					<li><a href="#">html</a></li>
+					<li><a href="#">bootstrap</a></li>
+					<li><a href="#">Ruby</a></li>
+				</ul>
+			  </div>
+			  <button class="pjt-button">See Project</button>
+			</div>
+            <div class="project-card">
+			  <span class="pjt-card-header heading">Profesional Art Printing Data</span>
+              <span class="pjt-card-sub-header sub-heading">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</span>
+			  <div >
+				<ul class="stack-list">
+					<li><a href="#">html</a></li>
+					<li><a href="#">bootstrap</a></li>
+					<li><a href="#">Ruby</a></li>
+				</ul>
+			  </div>
+			  <button class="pjt-button">See Project</button>
+			</div>
+            <div class="project-card">
+			  <span class="pjt-card-header heading">Profesional Art Printing Data</span>
+              <span class="pjt-card-sub-header sub-heading">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</span>
+			  <div >
+				<ul class="stack-list">
+					<li><a href="#">html</a></li>
+					<li><a href="#">bootstrap</a></li>
+					<li><a href="#">Ruby</a></li>
+				</ul>
+			  </div>
+			  <button class="pjt-button">See Project</button>
+			</div>
+            
+            
+          </div>
+        </section>
+        <section class="about">
+
+		  <div class="about-me">
+		    <div class="heading abt-header">About me</div>
```

Comment:
Kindly change this to a header  tag`hx that is h1-h6` This is for correct sematics and for screen readers to understand your app.

Reply:
done @Nyame-Wolf